Delivered-To: raj.khem@gmail.com
Received: by 10.90.86.4 with SMTP id j4cs313299agb;
        Sat, 8 Jan 2011 16:45:16 -0800 (PST)
Received: by 10.227.134.5 with SMTP id h5mr1312651wbt.75.1294533915992;
        Sat, 08 Jan 2011 16:45:15 -0800 (PST)
Return-Path: <yann.morin.1998@anciens.enib.fr>
Received: from smtp.smtpout.orange.fr (smtp01.smtpout.orange.fr [80.12.242.123])
        by mx.google.com with ESMTP id m42si33753731wej.65.2011.01.08.16.45.15;
        Sat, 08 Jan 2011 16:45:15 -0800 (PST)
Received-SPF: neutral (google.com: 80.12.242.123 is neither permitted nor denied by best guess record for domain of yann.morin.1998@anciens.enib.fr) client-ip=80.12.242.123;
Authentication-Results: mx.google.com; spf=neutral (google.com: 80.12.242.123 is neither permitted nor denied by best guess record for domain of yann.morin.1998@anciens.enib.fr) smtp.mail=yann.morin.1998@anciens.enib.fr
Received: from roazhon.bzh.lan ([90.32.245.227])
	by mwinf5d24 with ME
	id tClC1f0024v5z3u03ClEDA; Sun, 09 Jan 2011 01:45:15 +0100
From: "Yann E. MORIN" <yann.morin.1998@anciens.enib.fr>
To: uclibc@uclibc.org
Cc: Khem Raj <raj.khem@gmail.com>,
	Bernhard Reutner-Fischer <rep.dot.nop@gmail.com>,
	Carmelo AMOROSO <carmelo.amoroso@st.com>
Subject: [PATCH 1/7] ARM: reorder "Use BX" option
Date: Sun,  9 Jan 2011 01:45:04 +0100
Message-Id: <1294533910-19305-2-git-send-email-yann.morin.1998@anciens.enib.fr>
X-Mailer: git-send-email 1.7.1
In-Reply-To: <1294533910-19305-1-git-send-email-yann.morin.1998@anciens.enib.fr>
References: <1294533910-19305-1-git-send-email-yann.morin.1998@anciens.enib.fr>

"Use BX" is not available on all CPUs, so the option depends on
a correct CPU to be chosen . It is weird that e BX" then appears
_above_ the CPU selection, not below.

Move the "Use BX" after the CPU selection.

Signed-off-by: "Yann E. MORIN" <yann.morin.1998@anciens.enib.fr>
Cc: Khem Raj <raj.khem@gmail.com>
Cc: Bernhard Reutner-Fischer <rep.dot.nop@gmail.com>
Cc: Carmelo AMOROSO <carmelo.amoroso@st.com>
---
 extra/Configs/Config.arm |   14 +++++++-------
 1 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/extra/Configs/Config.arm b/extra/Configs/Config.arm
index b060ace..3b90e67 100644
--- a/extra/Configs/Config.arm
+++ b/extra/Configs/Config.arm
@@ -30,13 +30,6 @@ config CONFIG_ARM_EABI
 
 endchoice
 
-config USE_BX
-	bool "Use BX in function return"
-	default y
-	depends on !CONFIG_GENERIC_ARM && !CONFIG_ARM610 && !CONFIG_ARM710
-	help
-	  Use BX instruction for THUMB aware architectures.
-
 choice
 	prompt "Target Processor Type"
 	default CONFIG_GENERIC_ARM
@@ -131,3 +124,10 @@ config CONFIG_ARM_IWMMXT
 	select ARCH_HAS_MMU
 
 endchoice
+
+config USE_BX
+	bool "Use BX in function return"
+	default y
+	depends on !CONFIG_GENERIC_ARM && !CONFIG_ARM610 && !CONFIG_ARM710
+	help
+	  Use BX instruction for THUMB aware architectures.
-- 
1.7.1

